packages/ai-jsx/src/core/completion.tsx

```diff
@@ -203,6 +204,20 @@ export function AssistantMessage({ children }: { children: Node }) {
   return children;
 }
 
+export function ConversationHistory({ messages }: { messages: ChatCompletionResponseMessage[] }) {
+  return messages.map((message) => {
+    if (message.role === 'assistant') {
+      return <AssistantMessage>{message.content}</AssistantMessage>;
+    }
+    if (message.role === 'user') {
+      return <UserMessage>{message.content}</UserMessage>;
```

Comment:
Assign name as well

Reply:
I see `name` on `UserMessage` but I'm not sure what it means or what it would be used for.

Reply:
You can assign names to users in the conversation history to make it clear to the model that there are different users. Assuming the JSON type you're mapping is from OpenAI it should be there too?

Reply:
Hmm, I don't see that in the type. But maybe that's because it's the `openai-edge` type which lags behind.